so/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Find the numpy include path
```

Comment:
Make this module build conditionally on finding Bzip2 and FLAC, since these are technically optional libraries in core.